app/forms/otp_verification_form.rb

```diff
@@ -0,0 +1,22 @@
+class OtpVerificationForm
+  def initialize(user, code)
+    @user = user
+    @code = code
+  end
+
+  def submit
+    @success = user.authenticate_direct_otp(code)
```

Comment:
We want to keep `result` as a separate method in case we want to add more properties to it, but I can move the first line to a method.